### PR TITLE
feat: public fork

### PR DESCRIPTION
**Pull request by rabyyuson**
### Ticket

- https://horizenlabs.atlassian.net/browse/TOOLS-2591

### Changes

- rebase [private](https://github.com/HorizenLabs/sc_evm_blockscout) onto [public](https://github.com/HorizenLabs/eon-explorer) Blockscout

### Preview

https://github.com/HorizenLabs/eon-explorer/assets/5024180/02dad0b8-3819-433b-9abb-1e0da16e8325

### CODE REVIEW

File: apps/block_scout_web/assets/static/images/MetaData_img_blueprint_dune-blockex.jpg
Comment (yurikoinaba) [inline apps/block_scout_web/assets/static/images/MetaData_img_blueprint_dune-blockex.jpg:1]: We don't need this file anymore

**Review by yurikoinaba — APPROVED** [review]
The PR looks good to me, though the more recent commits, for forward transfers and fee payments, are not included yet.

**Comment by voyenavion** [comment]
Uploading Screen Recording 2023-08-10 at 2.33.33 PM.mov…

**Comment by voyenavion** [comment]
https://github.com/HorizenLabs/eon-explorer/assets/14279989/8de848f7-3ad0-49a8-98b4-85bb02b222ee

**Review by voyenavion — COMMENTED** [review]
lgtm

**Pull request merged**